### PR TITLE
LPS-190602 As an object admin, I want to define the output field of my object validation

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://liferay.atlassian.net/browse/LPS-190602

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:stable

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java [hunk 1]
```diff
@@ -254,6 +276,63 @@ public void validate(
...
+
+					break;
+				}
+			}
+
+			if (objectValidationRuleSetting == null) {
+				_objectValidationRuleSettingPersistence.remove(
+					oldObjectValidationRuleSetting);
+			}
+		}
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java]: Done! Change made in this [commit](https://github.com/liferay-objects/liferay-portal/pull/2356/commits/de43edc27547be5bb9f2ad0062d1af0c55009203#diff-640330ed21a9efd55d2476fb80feb4db0b1b4ddecdbaeeca377d80ace0209be5L186).

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java [hunk 2]
```diff
@@ -254,6 +276,63 @@ public void validate(
...
+
+		return TransformUtil.unsafeTransform(
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java]: Done! Change made in this [commit](https://github.com/liferay-objects/liferay-portal/pull/2356/commits/de43edc27547be5bb9f2ad0062d1af0c55009203#diff-640330ed21a9efd55d2476fb80feb4db0b1b4ddecdbaeeca377d80ace0209be5R282).

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**